.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Fix of issue(s): 
```

Comment:
I do not think we need this as issue number is mentioned in the commit message.

Reply:
As a practice i have seen most of us adding this field in the pr description. So i kept this as a option. IMO keeping this will increase the look and feel of pr template. WDYT ?

Reply:
@amitkrout Check this https://help.github.com/articles/closing-issues-using-keywords/ 
The advantge of adding keywords in the PR is that it automatically closes the issue when the PR gets merged. So it that sense I am ok with including it.

Reply:
@amitkrout In current form i.e. `Fix of issue(s): ` it will not auto close issues as the format <key from below list> \#issue number. Also semicolon is not expected between keyword and issue number. 

```
    close
    closes
    closed
    fix
    fixes
    fixed
    resolve
    resolves
    resolved
```

Reply:
Done